iphone/Classes/TiUIListView.m

```diff
@@ -39,9 +39,8 @@ @implementation TiUIListView {
     TiUIRefreshControlProxy* _refreshControlProxy;
 #endif
 
-    TiUISearchBarProxy *searchViewProxy;
-    UITableViewController *tableController;
```

Comment:
In UISearchController there is option to pass nil for result controller. In this case it will use the current tableview for showing the search result. We can not use different controller to show the result because it will create similar problem which is mentioned in TIMOB-16622 and TIMOB-16826. So there is no need of tableViewController.
searchViewProxy is there.